src/current/cockroachcloud/take-and-restore-customer-owned-backups.md

```diff
@@ -5,65 +5,197 @@ toc: true
+This page describes how to take and restore [_customer-owned backups_]({% link cockroachcloud/backup-and-restore-overview.md %}) from CockroachDB {{ site.data.products.serverless }} and CockroachDB {{ site.data.products.dedicated }} clusters.
 
-The examples on this page demonstrate how to back up and restore from cloud storage and `userfile` storage. We recommend using cloud storage for your backups.
+The [examples](#examples) on this page provide a quick overview of the backup features you can run to your own storage bucket. For more technical detail on the complete list of backup features, refer to:
 
-For a list of other backup types you can run manually, including [incremental](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-full-and-incremental-backups) and [encrypted](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-and-restore-encrypted-backups) backups, refer to the [Backup and Restore Overview](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/backup-and-restore-overview) page.
+- [Full backup](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-full-and-incremental-backups#full-backups)
+- [Incremental backup](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-full-and-incremental-backups#incremental-backups)
+- [Scheduled backup](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}//manage-a-backup-schedule)
+- [Backups with revision history](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-backups-with-revision-history-and-restore-from-a-point-in-time)
+- [Point-in-time restore](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-backups-with-revision-history-and-restore-from-a-point-in-time)
+- [Encrypted backup and restore](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-and-restore-encrypted-backups)
+- [Locality-aware backup and restore](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-and-restore-locality-aware-backups)
+- [Locality-restricted backup execution](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-locality-restricted-backups)
+
+## Examples
+
+Before you begin, connect to your cluster. For guidance on connecting to your CockroachDB {{ site.data.products.cloud }} cluster, visit [Connect to a CockroachDB {{ site.data.products.serverless }} Cluster]({% link cockroachcloud/connect-to-a-serverless-cluster.md %}) or [Connect to Your CockroachDB {{ site.data.products.dedicated }} Cluster]({% link cockroachcloud/connect-to-your-cluster.md %}).
+
+The examples on this page demonstrate how to back up and restore from your own storage bucket.
+
+### Full backup
```

Comment:
The examples in this section are all copied from the self-hosted section. Not new. It would be good to review structure.